web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -523,6 +523,31 @@ function ClusterFilterItem(props: SearchResultItem<SearchResultCluster>) {
   );
 }
 
+function DisplayResultsItem(props: SearchResultItem<DisplayResults>) {
+  return (
+    <IconAndContent Icon={icons.Magnifier} iconColor="text.slightlyMuted">
+      <Text typography="body1">
+        {' '}
+        Display {props.searchResult.value ? 'search' : 'all'} results{' '}
+        {props.searchResult.value && (
+          <>
+            for{' '}
+            <strong>
+              <Highlight
+                keywords={[props.searchResult.value]}
+                text={props.searchResult.value}
+              />
```

Comment:
This throws an exception when searching for `labels[`. hightlight-words-core.findAll seems to take the input and construct some kind of a regex from it. Passing `autoEscape: true` to `findAll` in `Highlight` fixes this problem.